docs/scripts/generate_notebooks.py

```diff
@@ -22,8 +23,12 @@ def make_header(notebook_path):
 
 
 def export_notebook(notebook_path, output_path):
-    exporter = MarkdownExporter()
-    output, resources = exporter.from_filename(notebook_path)
+
+    notebook_exporter = NotebookExporter()
+    notebook_exporter.preprocessors = [ClearOutputPreprocessor()]
+    notebook_content, _ = notebook_exporter.from_filename(notebook_path)
+    markdown_exporter = MarkdownExporter()
+    output, _ = markdown_exporter.from_notebook_node(notebook_content)
```

Comment:
@scottire there seems to be some error in the `notebook_content` you're passing to `markdown_exporter.from_notebook_node(notebook_content)`. This is the error I'm getting when I run it: 

```bash
Traceback (most recent call last):
  File "/Users/niware_wb/Documents/wandb_internal/smle/demos/llms/weave/weave_source/docs/./scripts/generate_notebooks.py", line 67, in <module>
    main()
  File "/Users/niware_wb/Documents/wandb_internal/smle/demos/llms/weave/weave_source/docs/./scripts/generate_notebooks.py", line 60, in main
    export_all_notebooks_in_primary_dir()
  File "/Users/niware_wb/Documents/wandb_internal/smle/demos/llms/weave/weave_source/docs/./scripts/generate_notebooks.py", line 53, in export_all_notebooks_in_primary_dir
    export_notebook(
  File "/Users/niware_wb/Documents/wandb_internal/smle/demos/llms/weave/weave_source/docs/./scripts/generate_notebooks.py", line 31, in export_notebook
    output, _ = markdown_exporter.from_notebook_node(notebook_content)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/niware_wb/Documents/wandb_internal/smle/demos/llms/weave/weave_source/venv/lib/python3.11/site-packages/nbconvert/exporters/templateexporter.py", line 408, in from_notebook_node
    nb_copy, resources = super().from_notebook_node(nb, resources, **kw)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/niware_wb/Documents/wandb_internal/smle/demos/llms/weave/weave_source/venv/lib/python3.11/site-packages/nbconvert/exporters/exporter.py", line 150, in from_notebook_node
    if "language" in nb["metadata"]:
                     ~~^^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
```